docs/guide/integrations.rst

```diff
@@ -0,0 +1,49 @@
+.. _integrations:
+
+============
+Integrations
+============
+
+Weights & Biases
+================
+
+Weights & Biases provides a callback for experiment tracking that allows to visualize and share results.
+
+The full documentation is available here: https://docs.wandb.ai/guides/integrations/other/stable-baselines-3
+
+.. code-block:: python
+
+  import gym
+  import wandb
+  from wandb.integration.sb3 import WandbCallback
+
+  from stable_baselines3 import PPO
+
+  config = {
+      "policy_type": "MlpPolicy",
+      "total_timesteps": 25000,
+      "env_name": "CartPole-v1",
+  }
+  run = wandb.init(
+      project="sb3",
+      config=config,
+      sync_tensorboard=True,  # auto-upload sb3's tensorboard metrics
+      # monitor_gym=True,  # auto-upload the videos of agents playing the game
```

Comment:
any reason to comment out this line?

Reply:
yep, to keep the example minimal (video recording is usually slow and not available for every env)

Reply:
we link to W&B doc for more complete example

Reply:
That makes sense